apps/aehttp/src/aehttp_dispatch_ext.erl

```diff
@@ -189,17 +189,11 @@ empty_fields_in_genesis() ->
 % if this changes - both functions should be changes:
 % empty_fields_in_genesis/0 and values_for_empty_fields_in_genesis/0
 values_for_empty_fields_in_genesis() ->
```

Comment:
This function mimics the behaviour of `aec_blocks:serialize_to_map/1` for the fields at hand.
I was looking for a _proper_ way of creating an inproper `#block{}` (having only those fields) but I'd rather not polute aec_blocks' interface.